runtime/src/tests/functionalTests/support/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './core'
```

Comment:
Bad structure.

In this case, why not just rename `core.ts` to `index.ts`?

Reply:
We might add more modules to the support package. Right now we have `core` and `constants` in the support package.
I'll update index.ts to export `core` and `constants`.

I've kept the export in this file so that we can do 
```
import * as support from './support'

...
support.Constants()
support.writeScreenshot()
...
```